lib_cxng/include/lcx_aes.h

```diff
@@ -56,27 +56,12 @@
  */
 cx_err_t cx_aes_init_key_no_throw(const uint8_t *rawkey, size_t key_len, cx_aes_key_t *key);
 
+
 /**
- * @brief   Initializes an AES Key.
- *
- * @details Once initialized, the key can be stored in non-volatile memory
- *          and directly used for any AES processing.
- *          This function throws an exception if the initialization fails.
- *
- * @warning It is recommended to use #cx_aes_init_key_no_throw rather
- *          than this function.
- *
- * @param[in]  rawkey  Pointer to the supplied key.
- *
- * @param[in]  key_len Length of the key: 16, 24 or 32 octets.
- *
- * @param[out] key     Pointer to the key structure. This must not be NULL.
- *
- * @return             Length of the key.
- *
- * @throw              CX_INVALID_PARAMETER
+ * @deprecated
+ * See #cx_aes_init_key_no_throw
  */
-static inline size_t cx_aes_init_key ( const unsigned char * rawkey, unsigned int key_len, cx_aes_key_t * key )
+__attribute__((deprecated)) static inline size_t cx_aes_init_key ( const unsigned char * rawkey, unsigned int key_len, cx_aes_key_t * key )
```

Comment:
could we change this to be a decorator as done here: https://github.com/LedgerHQ/ledger-secure-sdk/commit/27c6ccd2f05c73f405b2e70497cf4a422fd2df8f
This should allow to easily disable / change it depending on compiler or build target or whatever.

Reply:
ok